user-guide.html

```diff
@@ -118,39 +121,39 @@ <h5 class="modal-title font-head" id="exampleModalLongTitle">Data catalogue</h5>
 
     <!-- Alternative content block -->
     <div class="row" id="subhead">
-      <div class="col-sm-2"></div><!-- Spacing col -->
+      <div class="col-xl-2"></div><!-- Spacing col -->
 
       <!-- Left content -->
-      <div class="col-sm mainContent">
+      <div class="col-xl-5 mainContent">
         <div class="content-container">
           <h1 class="font-head-light">User guide</h1>
           <br>
           <h4 id="subheading">This is a short guide to the dashboard and how to use it 
             to identify vulnerable communities in Wales
             <br><br>
             A full guide can be found in the <a href="https://github.com/DynamicGenetics/Covid-Communities-Map/wiki">Wiki</a> 
```

Comment:
Suggested wording change: 
"Full documentation on our methodology and datasources can be found..."

Reply:
Thank you again, nicer wording- done.